### ISSUE TITLE
MeasureReport Definitions

### ISSUE THREAD

**Issue by jawalonoski** [AudaciousInquiry/fhir-saner#75]
It would be nice if the MeasureReport population variables/codes could be presented in a Table so that the definitions are easier to find/read.

This example table is not great (maybe it is bad), but it gives an idea of what I'm looking for:

Group System | Group Code | Population System | Population Code | Definition
-------|------------|--------|------|--------------
`http://hl7.org/fhir/us/saner/CodeSystem/MeasureGroupSystem` | `Beds` | `http://hl7.org/fhir/us/saner/CodeSystem/MeasurePopulationSystem`<br/>`http://terminology.hl7.org/CodeSystem/measure-population` | `numTotBeds`<br/>`initial-population` | Total Beds
 |  |  | `http://hl7.org/fhir/us/saner/CodeSystem/MeasurePopulationSystem`<br/>`http://terminology.hl7.org/CodeSystem/measure-population` | `numBeds`<br/>`measure-population` | Inpatient Beds

**Comment by GinoCanessa** [comment]
First pass [here](https://github.com/microsoft-healthcare-madison/learning-spike-erp/blob/master/markdown/measures.md).  Thoughts?

**Comment by jawalonoski** [comment]
Yes, this looks good. This structure helps a lot. Makes it easier to digest.

**Comment by keithboone** [comment]
This is awesome.

I'm going to:

1. Incorporate these into the _intro.md pages for existing measures immediately in the next build.
2. Figure out how to automatically generate this from existing measure data in FSH files as part
of the build process (when I get a chance, not during connectathon).

**Comment by keithboone** [comment]
Structure as table with Group/Population[/Stratifier] and then System, and then maybe some text explaining the content.

Integrate this into IG build from Measure Definition, transforming from Resource, either using XSLT, Liquid or some other mechanism.
